build/android/app/build.gradle

```diff
@@ -75,10 +75,13 @@ task prepareAssets() {
 	copy {
 		from "${projRoot}/games/${gameToCopy}" into "${assetsFolder}/games/${gameToCopy}"
 	}
-	/*copy {
-		// ToDo: fix broken locales
-		from "${projRoot}/po" into "${assetsFolder}/po"
-	}*/
+	fileTree("${projRoot}/po").include("**/*.po").forEach { poFile ->
+		def moPath = "${assetsFolder}/locale/${poFile.parentFile.name}/LC_MESSAGES/"
+		file(moPath).mkdirs()
+		exec {
+			commandLine 'msgfmt', '-o', "${moPath}/minetest.mo", poFile
+		}
+	}
```

Comment:
There should be a comment that indicates that gradle will not be able to do this on Windows at all, as well as on macOS without `brew install gettext`!

Reply:
True. Is there a place to add  this note ?

Reply:
@sfan5 your suggestions?

Reply:
Windows can do that as long `msgfmt` is within the path. All it needs is a comment stating that this section requires `gettext` to be installed and available in PATH.

Reply:
I'll see if I can add a test and break early with a meaningful error message.